impl/src/test/java/org/ehcache/docs/GettingStarted.java

```diff
@@ -362,6 +362,29 @@ public void cacheSerializingCopiers() throws Exception {
   }
 
   @Test
+  public void cacheCopierAtCacheManagerLevel() {
+    DefaultCopyProviderConfiguration copierType = new DefaultCopyProviderConfiguration()
+        .addCopierFor(Person.class, (Class)SerializingCopier.class);
```

Comment:
I think the issue for #605 was created with a `Serializable` implementation of `Description`. He wasn't using the `Description` class we have in this `GettingStarted` document. But for issue #604 he used a non-serializable `Description` implementation.